src/pages/settings/custom-fields/invoices/Invoices.tsx

```diff
@@ -18,11 +22,15 @@ import { Field } from '../components';
 
 export function Invoices() {
   const [t] = useTranslation();
+
+  const disabledCustomFields = !proPlan() && !enterprisePlan() && isHosted();
+
   const pages = [
     { name: t('settings'), href: '/settings' },
     { name: t('custom_fields'), href: '/settings/custom_fields' },
     { name: t('invoices'), href: '/settings/custom_fields/invoices' },
   ];
+
   useEffect(() => {
     document.title = `${import.meta.env.VITE_APP_TITLE}: ${t('custom_fields')}`;
   });
```

Comment:
Can we refactor this to use `useTitle` hook?

Reply:
Sure, refactored!